ci/scripts/test_cml.sh

```diff
@@ -180,19 +176,12 @@ INSTALLED_CP=`pip freeze | grep "concrete-python"`
 echo
 echo "Installed Concrete-Python: ${INSTALLED_CP}"
 
-# Launch CML pytests
+# Launch CML tests with pytest (and ignore flaky ones)
+# As compared to regular `make pytest`, known flaky errors from Concrete ML are simply ignored
+# and coverage is disabled
+# The "-x" option is added so that the run stops at the first test that fails
 echo
-echo "Launching CML tests"
+echo "Launching CML tests (no flaky)"
+echo "make pytest_no_flaky PYTEST_OPTIONS="-x""
+make pytest_no_flaky PYTEST_OPTIONS="-x"
```

Comment:
I'm not sure we want to stop at the first test that fails, maybe we want to know what are all the tests that are failing in CML because of the changes

Reply:
yes, your choice! I added the -x because I thought you wanted to have the failure state as soon as possible, but if you prefer to see all the errors, fine too. It's about 1h long test

Reply:
@youben11 should I remove it ? should I make it optional ?